bcda/api_test.go

```diff
@@ -135,7 +135,8 @@ func (s *APITestSuite) TestBulkEOBRequestUserDoesNotExist() {
 	acoID := "dbbd1ce1-ae24-435c-807d-ed45953077d3"
 	userID := "82503a18-bf3b-436d-ba7b-bae09b7ffdff"
 	tokenID := "665341c9-7d0c-4844-b66f-5910d9d0822f"
-	ad := auth.AuthData{ACOID: acoID, UserID: userID, TokenID: tokenID}
+	ad := auth.AuthData{ACOID: acoID, UserID: userID, TokenID: tokenID,}
```

Comment:
why this change?  idt its necessary unless it spans multiple lines

Reply:
You're right.